.travis.yml

```diff
@@ -12,7 +12,7 @@ before_install:
 - npm install -g yarn@latest
 install:
 - yarn install --frozen-lockfile
-- yarn run bootstrap
+- yarn run bootstrap --ignore textlint-script-release
```

Comment:
I want to avoid this error, but this does not work...

> error inquirer@5.0.1: The engine "node" is incompatible with this module. Expected version ">=6.0.0".